styles/index.css

```diff
@@ -22,23 +22,37 @@ html {
   */
 html,
 body {
-  height: 100%;
+  height: 200%;
```

Comment:
is there a reason why you made this 200%

Reply:
i dont remember making it 200% tho
but when i tried making it 100% to know why it was 200 the page was shorter so the form looked as if it continued but page height wasnt enough,it also made the box shadow for the bottom not show.ill drop screenshots of it as 100 and 200
```suggestion
  height: 200%;
![Screenshot (300)](https://user-images.githubusercontent.com/89720943/202403338-9b1093e2-c495-454e-9f6a-1b14944fe505.png)
![Screenshot (299)](https://user-images.githubusercontent.com/89720943/202403360-90be8aa3-c6c2-44cf-9ebf-464292db2109.png)

```